.github/sha.json

```diff
@@ -1,6 +1,6 @@
 {
-  "lib/ansible/module_utils/azure_rm_common.py": "1111111111111111111111111111111111111111", 
-  "lib/ansible/modules/cloud/azure": "1111111111111111111111111111111111111111", 
-  "packaging/requirements/requirements-azure.txt": "1111111111111111111111111111111111111111", 
-  "test/integration/targets": "1111111111111111111111111111111111111111"
+  "lib/ansible/module_utils/azure_rm_common.py": "c29fe66edf7972dc8dd49f6216af2a2cfbe6add4", 
+  "lib/ansible/modules/cloud/azure": "133b70ff4503be52b007d3abe642d7d7f360b686", 
```

Comment:
should this file be actually checked in?

Reply:
yes, it's used to save where's last commit synced